src/editors/containers/ProblemEditor/components/EditProblemView/AnswerWidget/AnswersContainer.jsx

```diff
@@ -14,10 +16,9 @@ export const hooks = {
 }
 export const AnswersContainer = ({
   //Redux
-  problemType,
   answers
 }) => {
-
+  const problemType = useSelector(selectors.problem.problemType);
```

Comment:
Note we use `mapStateToProps` for selectors, as it allows us to easily write snapshot level components.

Reply:
Ack, I will make this change.